rocky/rocky/views/mixins.py

```diff
@@ -99,11 +106,11 @@ def get_origins(
         self,
         reference: Reference,
         organization: Organization,
-    ) -> tuple[list[OriginData], list[OriginData], list[OriginData]]:
+    ) -> Origins:
         declarations: list[OriginData] = []
         observations: list[OriginData] = []
         inferences: list[OriginData] = []
-        results = declarations, observations, inferences
+        results = Origins({"declarations": declarations, "observations": observations, "inferences": inferences})
```

Comment:
Looks good. I would only suggest using this dict as a literal instead of instantiating the `TypedDict`. This will ensure consistency with the rest of the code base and aligns with the intended use of `TypedDict`. The current example is harder to read, as it implies that `Origins` is a class/ function/ data model that takes a single dictionary argument, rather than a dictionary itself

```suggestion
        results = {"declarations": declarations, "observations": observations, "inferences": inferences}
```